descriptor.go

```diff
@@ -86,6 +87,10 @@ const (
 	VBIDataServiceIDWSS                  = 0x5
 )
 
+const (
+	DescriptorHeaderSizeBytes = 2
```

Comment:
Is there any gain to export this constant?

Reply:
it's even better - we don't even need this constant at all